### DISCUSSION TITLE
The project page showed blank

### DISCUSSION THREAD

**Discussion by yasiryagi** [subquery/subql#1007] [Frontier TestNet - Bug Reporting]
The project page showed blank on the indexer admin
![15](https://user-images.githubusercontent.com/4862448/167710714-c215662a-2653-4801-80bb-63d84490c6b1.PNG)

**Comment by mzxyz** [comment]
Can you confirm that the coordinator service version is `v0.19.0`? We publish a new version to fix this issue.

**Comment by yasiryagi** [comment]
Upgraded to 1.9. and fixed.
